packaging/tags.py

```diff
@@ -652,11 +652,28 @@ def _linux_platforms(is_32bit=_32_BIT_INTERPRETER):
             linux = "linux_armv7l"
     manylinux_support = []
     _, arch = linux.split("_", 1)
+    pep600_support = []
     if _have_compatible_manylinux_abi(arch):
         if arch in {"x86_64", "i686", "aarch64", "armv7l", "ppc64", "ppc64le", "s390x"}:
             manylinux_support.append(
                 ("manylinux2014", (2, 17))
             )  # CentOS 7 w/ glibc 2.17 (PEP 599)
+            if arch in ("x86_64", "i686"):
+                # glibc in manylinux1 is 2.5
+                min_minor = 5
+            else:
+                # glibc in manylinux2014 is 2.5
+                min_minor = 17
+            for glibc_minor in range(min_minor, 50):
+                # PEP 600 - up to glibc 2.50 (as of April 2020 last is 2.31)
```

Comment:
50 is rather arbitrary, any thoughts of a better stopping mechanism? Would parsing `_glibc_version_string()` be better?